apps/web/components/shared/qa-server.tsx

```
@@ -0,0 +1,136 @@
+    .from("comments")
+    .select("id, content, created_at, project_id, author_id, type, parent_comment_id, is_resolved")
+    .eq("project_id", projectId)
+    .eq("type", "question")
+    .is("parent_comment_id", null)
+    .order("created_at", { ascending: false });
+
+  
+  let questionsWithAuthors: CommentData[] = [];
+  
+  if (initialQuestions && initialQuestions.length > 0) {
+    const authorIds = [...new Set(initialQuestions.map((item: any) => item.author_id))];
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)